### ISSUE TITLE
fitbit app

### ISSUE THREAD

**Issue by tenderitaf** [Talos-crypto/Talos-Android#2]
Hi,
It worth to mention that the fitbit app should be registred as personal app in order to get intra days activtities.

Regards

**Comment by lubux** [comment]
Yes, you are right. Fitbit has privacy restrictions for high-resolution data from its API, that is why a personal account is required
